downloads/swan-lake-release-notes/swan-lake-beta1/RELEASE_NOTE.md

```diff
@@ -1,46 +1,219 @@
+  | T1 & T2                   | left          |
+  | T1 &#124; T2              | left          |
+  | function (args) returns T | right         |
 
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
+## Runtime Updates
+
+### Improvements
+
+#### Improved Configurable Variables to Support Enum Types
+
+Configurable variables with enum types are supported to provide values through command-line arguments and a TOML file.
+
+For example, see the configurable variable with the `enum` type.
+
+```ballerina
+public enum HttpVersion {
+    HTTP_1_1,
+    HTTP_2
+}
+
+configurable configLib:HttpVersion & readonly httpVersion = ?;
+```
+
+The value for `httpVersion` can be provided via the `Config.toml` file or as a command-line argument as shown below.
+
+```toml
+[configUnionTypes]
+httpVersion = "HTTP_1_1"
+```
+
+The command-line argument will be as shown below.
+
+```bash
+-ChttpVersion=HTTP_1_1
+```
+
+#### Improved Configurable Variables to Support Map Types
+
+The `configurable` feature is improved to support variables with the `map` types through the TOML syntax. 
+
+For example, if the `map`-typed configurable variables are defined in the following way, 
+
+``` ballerina
+configurable map<string> admin = ?;
+
+type HttpResponse record {|
+    string method;
+    string httpVersion = "HTTP_1_0";
+    map<string> headers;
+|};
+
+configurable HttpResponse response = ?;
+configurable map<string>[] users = ?;
+```
+
+the values can be provided in the `Config.toml` as follows.
+
+```toml
+[admin]
+username = "John Doe"
+mail = "John@hotmail.com"
+location = "LK"
+
+[response]
+method = "POST"
+httpVersion = "HTTP_2_0"
+headers.Status = "200 OK"
+headers.Content-Type = "text/html"
+
+[[users]]
+username = "Jack"
+location = "UK"
+
+[[users]]
+username = "Jane"
+location = "US"
+```
+
+### Breaking Changes
+
+- The `io.ballerina.runtime.api.types.Type#getName` and `io.ballerina.runtime.api.types.Type#getQualifiedName` methods now return an empty string if no name was associated with the type. The `io.ballerina.runtime.api.types.Type#toString` method can be used to get the string representation of a type if required.
+- `Wait` actions that wait for expressions that are not named workers can return errors now. The eventual type of such wait future expressions is now `T|error`in which `T` is the type of the original return value.
+
+### Bug Fixes
+
+To view bug fixes, see the [GitHub milestone for Swan Lake Beta1](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Beta1%22+label%3AType%2FBug+label%3ATeam%2FjBallerina).
+
+## Standard Library Updates
+
+### New Features
+
+#### `graphql` Package
+- Added declarative auth configurations
```

Comment:
Can't we just say
```suggestion
- Added declarative auth configuration
```

Reply:
It should be "Added *the* declarative auth configuration.". @ThisaruGuruge - Please advise.

Reply:
There are multiple configurations, therefore I think it should be configurations.

Reply:
Depending on the context I believe configuration is also used as the plural form. @praneesha, thoughts please?

Reply:
+1